discovery/bootstrapper.go

```diff
@@ -274,8 +274,9 @@ search:
 		for _, dnsSeed := range d.dnsSeeds {
 			// We'll first query the seed with an SRV record so we
 			// can obtain a random sample of the encoded public
-			// keys of nodes.
-			_, addrs, err := net.LookupSRV("nodes", "tcp", dnsSeed)
+			// keys of nodes. We use the lndLookupSRV function for
+			// this task.
+			_, addrs, err := d.lookupFns[1].(func(string, string, string) (string, []*net.SRV, error))("nodes", "tcp", dnsSeed)
```

Comment:
Why not just have Lookup host as a config field directly? This is much less obvious as written.

Reply:
Right